.travis.yml

```diff
@@ -28,6 +28,8 @@ install:
    - cd ~/catkin_ws
 
 before_script: # Use this to prepare your build for testing e.g. copy database configurations, environment variables, etc.
-  - source /opt/ros/$ROS_DISTRO/setup.bash  
+  - source /opt/ros/$ROS_DISTRO/setup.bash
 script: # All commands must exit with code 0 on success. Anything else is considered failure.
   - catkin_make -j2
+  - source ~/catkin_ws/devel/setup.bash # Allow installing dependencies (recognizing buzzmobile)
+  - rosdep install buzzmobile # Install dependencies for buzzmobile
```

Comment:
As discussed offline, you can only install deps after you source the `setup.bash`, which is in the `devel/`, which is generated from `catkin_make`. So the way this is done seems correct.

At one point, when we have rostests ( see #20 ) we will move these three script commands to `before_script` and have the test command be the `script`.